token/program-2022/src/extension/confidential_transfer/processor.rs

```diff
@@ -397,6 +398,12 @@ fn process_deposit(
     let mint_data = &mint_info.data.borrow_mut();
     let mint = PodStateWithExtensions::<PodMint>::unpack(mint_data)?;
 
+    if let Ok(extension) = mint.get_extension::<PausableConfig>() {
```

Comment:
Oh yeah, sorry, I forgot to mention this in the PR description. Do you think that makes sense? If the mint is paused, should people be able to move tokens between their confidential and non-confidential balance? I was thinking yes. I'll update the comment and also add tests

cc @gitteri @jnwng in case they want to chime in

Reply:
Yeah I think it makes sense either way. Way back when we first started out on token22, we were able to deposit and withdraw funds to other accounts, so deposit/withdraw were another way to transfer funds. We have since restricted deposit/withdraw to work within same accounts only, but some intuition from that time remains, which makes me feel that pausing deposits/withdraws makes sense. Given this, the current changes looks good to me!